### ISSUE TITLE
Feature Request: Add Firebase Authentication

### ISSUE THREAD

**Issue by Stelath** [huggingface/chat-ui#1287]
Hello, I was wondering how complicated it would be to add the ability to link this to a firebase app and then utilize firebase's auth service instead of the OpenID login? Thanks for any help and if this became a feature that would be awesome!

**Comment by nsarrazin** [comment]
Not sure this is something we want to tackle, as it would be nice to remain agnostic on the backend side. If there's strong demand maybe but so far most people manage with OpenID I think :eyes:

**Comment by Stelath** [comment]
Fair enough, I managed to hack together a reasonable solution for my case with Firebase, though as you said it destroyed the OpenID backend otherwise I would PR it.

** I'm sure there's a way to do it that would keep both working just fine, I just was short on time and didn't need the OpenID
